README.rst

```diff
@@ -63,10 +63,24 @@ Here's example of the things needed inside a Scrapy project's ``settings.py`` fi
 Usage
 -----
 
-Set the ``zyte_api`` `Request.meta
-<https://docs.scrapy.org/en/latest/topics/request-response.html#scrapy.http.Request.meta>`_
-key to download a request using Zyte API. Full list of parameters is provided in the
-`Zyte API Specification <https://docs.zyte.com/zyte-api/openapi.html#zyte-openapi-spec>`_.
+To enable every request to be sent through Zyte API, you can set the following
+in the ``settings.py`` file or `any other settings within Scrapy
+<https://docs.scrapy.org/en/latest/topics/settings.html#populating-the-settings>`_:
+
+.. code-block:: python
+
+    ZYTE_API_DEFAULT_PARAMS = {
+        "browserHtml": True,
+        "geolocation": "US",
+    }
+
+You can see the full list of parameters in the `Zyte API Specification
+<https://docs.zyte.com/zyte-api/openapi.html#zyte-openapi-spec>`_.
+
+On the other hand, you could also control it on a per request basis by setting the
+``zyte_api`` key in `Request.meta <https://docs.scrapy.org/en/latest/topics/request-response.html#scrapy.http.Request.meta>`_.
+When doing so, it will override any parameters that was set in the 
+``ZYTE_API_DEFAULT_PARAMS`` setting.
```

Comment:
Should we allow `meta["zyte_api"] = False` as a way to disable Zyte Data API for a specific request while using `ZYTE_API_DEFAULT_PARAMS`?

Reply:
The way [it's currently setup](https://github.com/scrapy-plugins/scrapy-zyte-api/blob/main/scrapy_zyte_api/handler.py#L51-L54), setting it as `False`, `None`, `0`, or other False-like value would disable Zyte API for the specific request.

I believe we should allow disabling it on a per request basis to support certain use cases. This would be especially useful when users start migrating their existing Scrapy projects from using SPM into Zyte API. For instance, `scrapy-zyte-smartproxy` uses `dont_proxy=True` ([reference](https://github.com/scrapy-plugins/scrapy-zyte-smartproxy/blob/master/scrapy_zyte_smartproxy/middleware.py#L292)) uses it.